templates/bundles/DemosPlanProcedureBundle/DemosPlanProcedure/public_detail.html.twig

```diff
@@ -124,21 +124,20 @@
 
                             {% apply spaceless %}
                                 {% if ownsProcedure(proceduresettings) %}
-                                    {# Display procedure phase of institutions AND citizens to enable planner user to quickly check correct settings #}
-                                    {{ getProcedurePhase(proceduresettings, 'internal') ~ ' / ' ~ getProcedurePhase(proceduresettings, 'public') }}
-                                {% else %}
-                                    {# Display procedure phase (depending on role) #}
-                                    {{ getProcedurePhase(proceduresettings) }}
+                                    {# Display procedure phase of institutions to enable planner user to quickly check correct settings #}
+                                    {{ getProcedurePhase(proceduresettings, 'internal') }}
+                                    {{- contextualHelp('help.public.detail.phase.'~ getProcedurePhaseKey(proceduresettings, 'internal'), ['o-tooltip--question-circle'|prefixClass, 'c-infolist__help u-ml-0_25'|prefixClass]) -}}
+                                    {{ '\n'|nl2br }}
                                 {% endif %}
 
                                 {# Display a "legal notice" for institutions #}
                                 {% if hasPermission('feature_procedure_legal_notice_read') and proceduresettings.settings.legalNotice|default != '' %}
                                     ({{ proceduresettings.settings.legalNotice }})
                                 {% endif %}
                             {% endapply %}
-
-                            {{- contextualHelp('help.public.detail.phase.'~ getProcedurePhaseKey(proceduresettings), ['o-tooltip--question-circle'|prefixClass, 'c-infolist__help u-ml-0_25'|prefixClass]) -}}
-
+                            {# Display public procedure phase without login or logged as citizen  #}
+                            {{ getProcedurePhase(proceduresettings, 'public') }}
+                            {{- contextualHelp('help.public.detail.phase.'~ getProcedurePhaseKey(proceduresettings, 'public'), ['o-tooltip--question-circle'|prefixClass, 'c-infolist__help u-ml-0_25'|prefixClass]) -}}
```

Comment:
I just wondered why the text and the tooltip where not in the spaceless-block before, but now and the second text with tooltip is completely without spaceless...

Reply:
thank you for the advice. i put it inside the spaceless block

Reply:
https://github.com/demos-europe/demosplan-core/pull/615/commits/75aded228884944d57772b3acbfba09113e24a61

Reply:
Removed spaceless block because its not necessary here https://github.com/demos-europe/demosplan-core/pull/615/commits/ccbfbbd7189dd79e47169e31031391ad76e75206